src/internal/expr.rs

```diff
@@ -144,13 +135,26 @@ impl Expr {
     }
 }
 
+/// Returns an expression that evaluates to true if the subexpression
+/// evaluates to false.
+///
+/// This method exists instead of the `std::ops::Not` trait to distinguish
```

Comment:
`Expr` intentionally doesn't implement `std::ops::Not`, because Rust uses that for both logical and bitwise not, but the `Expr` type needs to distinguish those two operations.

Reply:
Thank you, I understand. Done